tsconfig.json

```
@@ -5,6 +5,7 @@
     "esModuleInterop": true,                              
     "forceConsistentCasingInFileNames": true,          
     "strict": true,                                      
-    "skipLibCheck": true                               
+    "skipLibCheck": true,
```

Why we need this config?

---

I did some research about this point: 
- skipLibCheck may be the only solution in cases where the code doesn’t compile due to a dependency library. 
- when skipLibCheck is enabled, tsc will ignore all errors coming from declaration files (d.ts). Some types may be treated as any due to ignored errors. Valid types will still be checked.
- skipLibCheck may decrease compilation time significantly, especially in larger projects